src/commands/doctor.ts

```diff
@@ -95,9 +95,24 @@ module.exports = {
       expoVersion = "-"
       expoWorkflow = "not installed"
     }
-
     const expoInfo = [column1("expo"), column2(expoVersion), column3(expoWorkflow)]
 
+    let depExpoCLIInfo = null
+
+    try {
+      const expoCLIVersionOutput = await run("expo-cli --version", { trim: true })
+      if (expoCLIVersionOutput) {
+        const expoCLIVersion = expoCLIVersionOutput.replace("v", "")
+        depExpoCLIInfo = [
+          column1("expo global cli"),
+          column2(expoCLIVersion),
+          column3("Deprecated: Found 'expo-cli' installed. Please remove it."),
```

Comment:
```suggestion
          colors.yellow(column3("Deprecated: Found 'expo-cli' installed. Please remove it.")),
```

Suggestion for clarity to the user: This will highlight the warning better in the terminal, ie:
![image](https://github.com/infinitered/ignite/assets/374022/a37857c7-c852-4a59-be66-0c0b850eefe7)